bottles/frontend/ui/details-bottle.blp

```diff
@@ -230,6 +215,142 @@ template DetailsBottle : .AdwPreferencesPage {
 
   .AdwPreferencesGroup group_programs {
     title: _("Programs");
+
+    .AdwActionRow row_no_programs {
+      Box {
+        hexpand: true;
+
+        Label {
+          label: _("Use the Installers section or the \"Run executable\" button.");
+          wrap: false;
+
+          styles [
+            "dim-label",
+          ]
+        }
+      }
+    }
+    
+    .AdwPreferencesRow bottom_bar {
+      activatable: false;      
```

Comment:
activatable is false by default afaik

Reply:
this should likely have focusable: false though

Reply:
No, its true by default, need to be false else the whole bottom bar is highlighted on hover.
`focusable` does nothing as per my testing

Reply:
aabc0162f7230c82a404f8da9d709a41f314e791

Reply:
> No, its true by default, need to be false else the whole bottom bar is highlighted on hover.

It doesn't seem as such?